otx/algorithms/classification/tasks/inference.py

```diff
@@ -352,11 +352,18 @@ def _init_recipe(self):
 
         if train_type == TrainType.INCREMENTAL:
             recipe = os.path.join(recipe_root, "incremental.yaml")
+            if self._hyperparams.learning_parameters.enable_supcon:
+                recipe = os.path.join(recipe_root, "supcon.yaml")
```

Comment:
We don't have supcon.yaml. If you want to change hyperparameters for supcon, you should change template.yaml manually before training experiments.

Reply:
But we do have `supcon.yaml` alread in `feature/otx`:
https://github.com/openvinotoolkit/training_extensions/blob/feature/otx/otx/recipes/stages/classification/supcon.yaml

Reply:
Ah, sorry about that. I confused with template and recipe.

Reply:
No worries, it's very confusing.